Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Focus-center-absorbPointingOffsets.rst

```diff
@@ -0,0 +1,102 @@
+.. |author| replace:: *TDB*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _AT-On-sky-CWFS:
+
+##################################################################
+Beginning of the night: Focus, center and absorb pointing offsets
+##################################################################
+
+.. _Beginning_of_the_night-the-Telescope-Overview:
+
+Overview
+========
+
+This procedure must be followed by the observers before the beginning of dark time, prior to starting science or engineering activities.
+It will focus the telescope, center a target and absorb the pointing offsets. 
+
+.. _Beginning_of_the_night-Prerequisites:
+
+Prerequisites
+=============
+- The telescope is ready for on-sky operations: it has been vented (if applicable), mirror cover and dome shutter are open, AOS corrections are enabled. (:ref:`Prepare for on-sky <AT-Pre-Open-for-On-Sky-Operations>`)
```

Comment:
Rather than just put the link, maybe phrase it as:
The telescope is :ref:`open and ready for on-sky observations <AT-Pre-Open-for-On-Sky-Operations>`: it has....

Reply:
👍